api/v1beta1/hyperconverged_types.go

```diff
@@ -272,6 +272,12 @@ type HyperConvergedSpec struct {
 	// +kubebuilder:default=false
 	// +default=false
 	EnableApplicationAwareQuota *bool `json:"enableApplicationAwareQuota,omitempty"`
+
+	// LiveUpdateConfiguration holds defaults for live update features
+	// +optional
+	// +kubebuilder:default=false
+	// +default=false
```

Comment:
Please drop these two lines. The default value can't be `false` because `LiveUpdateConfiguration` is not a boolean field, and that why it breaks `make gogenerate`. 

Also, The zero value of a pointer is nil, so we don't need a default value here.